package com.simprints.id.exceptions.safe.secure

import com.simprints.id.exceptions.safe.SafeException


class AuthException(cause: Throwable) : SafeException(cause)
